Flaky Docker build due to ccache

Flaky build test: 
```
++ nproc

+ make -j72

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_main.c.d -g -O2 -Wall -W -Werror -c -o src/main.o src/main.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_args.c.d -g -O2 -Wall -W -Werror -c -o src/args.o src/args.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_ccache.c.d -g -O2 -Wall -W -Werror -c -o src/ccache.o src/ccache.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_cleanup.c.d -g -O2 -Wall -W -Werror -c -o src/cleanup.o src/cleanup.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_compopt.c.d -g -O2 -Wall -W -Werror -c -o src/compopt.o src/compopt.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_conf.c.d -g -O2 -Wall -W -Werror -c -o src/conf.o src/conf.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_counters.c.d -g -O2 -Wall -W -Werror -c -o src/counters.o src/counters.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_execute.c.d -g -O2 -Wall -W -Werror -c -o src/execute.o src/execute.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_exitfn.c.d -g -O2 -Wall -W -Werror -c -o src/exitfn.o src/exitfn.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_hash.c.d -g -O2 -Wall -W -Werror -c -o src/hash.o src/hash.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_hashutil.c.d -g -O2 -Wall -W -Werror -c -o src/hashutil.o src/hashutil.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_language.c.d -g -O2 -Wall -W -Werror -c -o src/language.o src/language.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_lockfile.c.d -g -O2 -Wall -W -Werror -c -o src/lockfile.o src/lockfile.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_manifest.c.d -g -O2 -Wall -W -Werror -c -o src/manifest.o src/manifest.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_mdfour.c.d -g -O2 -Wall -W -Werror -c -o src/mdfour.o src/mdfour.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_stats.c.d -g -O2 -Wall -W -Werror -c -o src/stats.o src/stats.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_unify.c.d -g -O2 -Wall -W -Werror -c -o src/unify.o src/unify.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_util.c.d -g -O2 -Wall -W -Werror -c -o src/util.o src/util.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_version.c.d -g -O2 -Wall -W -Werror -c -o src/version.o src/version.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_getopt_long.c.d -g -O2 -Wall -W -Werror -c -o src/getopt_long.o src/getopt_long.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_hashtable.c.d -g -O2 -Wall -W -Werror -c -o src/hashtable.o src/hashtable.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_hashtable_itr.c.d -g -O2 -Wall -W -Werror -c -o src/hashtable_itr.o src/hashtable_itr.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_murmurhashneutral2.c.d -g -O2 -Wall -W -Werror -c -o src/murmurhashneutral2.o src/murmurhashneutral2.c

gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_snprintf.c.d -g -O2 -Wall -W -Werror -c -o src/snprintf.o src/snprintf.c

gcc -g -O2 -Wall -W -Werror -o ccache src/main.o src/args.o src/ccache.o src/cleanup.o src/compopt.o src/conf.o src/counters.o src/execute.o src/exitfn.o src/hash.o src/hashutil.o src/language.o src/lockfile.o src/manifest.o src/mdfour.o src/stats.o src/unify.o src/util.o src/version.o src/getopt_long.o src/hashtable.o src/hashtable_itr.o src/murmurhashneutral2.o src/snprintf.o   -lm  -lz

+ make install

asciidoc -a revnumber=3.4.2 -d manpage -b docbook -o - doc/MANUAL.adoc | \

  perl -pe 's!<literal>(.*?)</literal>!<emphasis role="strong">\1</emphasis>!g' >doc/MANUAL.xml

a2x --doctype manpage --format manpage doc/MANUAL.xml

a2x: ERROR: "xsltproc"  --stringparam callout.graphics 0 --stringparam navig.graphics 0 --stringparam admon.textlabel 1 --stringparam admon.graphics 0  "/etc/asciidoc/docbook-xsl/manpage.xsl" "/work/deps/ccache/doc/MANUAL.xml" returned non-zero exit status 5

dev.mk:175: recipe for target 'doc/ccache.1' failed

make: *** [doc/ccache.1] Error 1

The command '/bin/sh -c /work/ubuntu_ccache.sh' returned a non-zero code: 2

Traceback (most recent call last):

  File "ci/build.py", line 347, in <module>

    sys.exit(main())

  File "ci/build.py", line 273, in main

    build_docker(platform, docker_binary, registry=args.docker_registry)

  File "ci/build.py", line 83, in build_docker

    check_call(cmd)

  File "/usr/lib/python3.5/subprocess.py", line 581, in check_call

    raise CalledProcessError(retcode, cmd)

subprocess.CalledProcessError: Command '['docker', 'build', '-f', 'docker/Dockerfile.build.jetson', '--build-arg', 'USER_ID=1001', '--cache-from', 'mxnetci/build.jetson', '-t', 'mxnetci/build.jetson', 'docker']' returned non-zero exit status 2

script returned exit code 1

```
http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/incubator-mxnet/detail/PR-11197/5/pipeline

Related log:

```
Step 6/26 : RUN /work/ubuntu_ccache.sh
 ---> Running in 1c1280d9d2f7
[91m+ pushd .
+ apt update
[0m/ /
[91m
WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

[0mHit:1 http://security.ubuntu.com/ubuntu xenial-security InRelease
Hit:2 http://archive.ubuntu.com/ubuntu xenial InRelease
Hit:3 http://archive.ubuntu.com/ubuntu xenial-updates InRelease
Hit:4 http://archive.ubuntu.com/ubuntu xenial-backports InRelease
Reading package lists...
Building dependency tree...
Reading state information...
All packages are up to date.
[91m+ apt install -y --no-install-recommends autoconf asciidoc xsltproc
[0m[91m
WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

[0mReading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  libsigsegv2 libxslt1.1 m4
Suggested packages:
  source-highlight vim-addon-manager autoconf-archive gnu-standards
  autoconf-doc libtool
Recommended packages:
  dblatex docbook-utils xmlto automake | automaken
The following NEW packages will be installed:
  asciidoc autoconf libsigsegv2 libxslt1.1 m4 xsltproc
0 upgraded, 6 newly installed, 0 to remove and 0 not upgraded.
Need to get 1392 kB of archives.
After this operation, 5391 kB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu xenial/main amd64 libsigsegv2 amd64 2.10-4 [14.1 kB]
Get:2 http://archive.ubuntu.com/ubuntu xenial/universe amd64 asciidoc all 8.6.9-3 [704 kB]
Get:3 http://archive.ubuntu.com/ubuntu xenial/main amd64 m4 amd64 1.4.17-5 [195 kB]
Get:4 http://archive.ubuntu.com/ubuntu xenial/main amd64 autoconf all 2.69-9 [321 kB]
Get:5 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libxslt1.1 amd64 1.1.28-2.1ubuntu0.1 [145 kB]
Get:6 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 xsltproc amd64 1.1.28-2.1ubuntu0.1 [13.4 kB]
[91mdebconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
[0m[91mdebconf: unable to initialize frontend: Readline
debconf: (This frontend requires a controlling tty.)
debconf: falling back to frontend: Teletype
[0m[91mdpkg-preconfigure: unable to re-open stdin: 
[0mFetched 1392 kB in 1s (934 kB/s)
Selecting previously unselected package libsigsegv2:amd64.
(Reading database ... 
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 26809 files and directories currently installed.)
Preparing to unpack .../libsigsegv2_2.10-4_amd64.deb ...
Unpacking libsigsegv2:amd64 (2.10-4) ...
Selecting previously unselected package asciidoc.
Preparing to unpack .../asciidoc_8.6.9-3_all.deb ...
Unpacking asciidoc (8.6.9-3) ...
Selecting previously unselected package m4.
Preparing to unpack .../archives/m4_1.4.17-5_amd64.deb ...
Unpacking m4 (1.4.17-5) ...
Selecting previously unselected package autoconf.
Preparing to unpack .../autoconf_2.69-9_all.deb ...
Unpacking autoconf (2.69-9) ...
Selecting previously unselected package libxslt1.1:amd64.
Preparing to unpack .../libxslt1.1_1.1.28-2.1ubuntu0.1_amd64.deb ...
Unpacking libxslt1.1:amd64 (1.1.28-2.1ubuntu0.1) ...
Selecting previously unselected package xsltproc.
Preparing to unpack .../xsltproc_1.1.28-2.1ubuntu0.1_amd64.deb ...
Unpacking xsltproc (1.1.28-2.1ubuntu0.1) ...
Processing triggers for man-db (2.7.5-1) ...
Processing triggers for libc-bin (2.23-0ubuntu10) ...
Setting up libsigsegv2:amd64 (2.10-4) ...
Setting up asciidoc (8.6.9-3) ...
Setting up m4 (1.4.17-5) ...
Setting up autoconf (2.69-9) ...
Setting up libxslt1.1:amd64 (1.1.28-2.1ubuntu0.1) ...
Setting up xsltproc (1.1.28-2.1ubuntu0.1) ...
Processing triggers for libc-bin (2.23-0ubuntu10) ...
[91m+ mkdir -p /work/deps
[0m[91m+ cd /work/deps
+ git clone --recursive -b v3.4.2 https://github.com/ccache/ccache.git
[0m[91mCloning into 'ccache'...
[0m[91mNote: checking out 'b1f509c3b2d492b182d3b892b67dae48c6c9d8b5'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

[0m[91m+ cd ccache
+ ./autogen.sh
[0mNow run ./configure and make
[91m+ ./configure
[0mconfigure: configuring ccache
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking for gcc option to accept ISO C99... none needed
checking how to run the C preprocessor... gcc -E
checking for a BSD-compatible install... /usr/bin/install -c
checking for ranlib... ranlib
checking for ar... ar
checking for bash... /bin/bash
checking for dirent.h that defines DIR... yes
checking for library containing opendir... none required
checking whether time.h and sys/time.h may both be included... yes
checking for sys/wait.h that is POSIX.1 compatible... yes
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for long long... yes
checking ctype.h usability... yes
checking ctype.h presence... yes
checking for ctype.h... yes
checking pwd.h usability... yes
checking pwd.h presence... yes
checking for pwd.h... yes
checking for stdlib.h... (cached) yes
checking for string.h... (cached) yes
checking for strings.h... (cached) yes
checking sys/time.h usability... yes
checking sys/time.h presence... yes
checking for sys/time.h... yes
checking sys/mman.h usability... yes
checking sys/mman.h presence... yes
checking for sys/mman.h... yes
checking termios.h usability... yes
checking termios.h presence... yes
checking for termios.h... yes
checking for gethostname... yes
checking for getopt_long... yes
checking for getpwuid... yes
checking for gettimeofday... yes
checking for mkstemp... yes
checking for realpath... yes
checking for strndup... yes
checking for strtok_r... yes
checking for unsetenv... yes
checking for utimes... yes
checking for compar_fn_t in stdlib.h... yes
checking for stdarg.h... yes
checking for varargs.h... no
checking for unsigned long long int... yes
checking for vsnprintf... yes
checking whether vsnprintf is C99 compliant... yes
checking for snprintf... yes
checking whether snprintf is C99 compliant... yes
checking for vasprintf... yes
checking for asprintf... yes
checking for library containing cos... -lm
checking for zlib >= 1.2.3... yes
checking for gzdopen in -lz... yes
checking whether byte ordering is bigendian... no
checking for inline... inline
checking for extern inline... no
configure: developer mode enabled
configure: creating ./config.status
config.status: creating dev.mk
config.status: creating Makefile
config.status: creating config.h
configure: now build ccache by running make
[91m++ nproc
[0m[91m+ make -j72
[0mgcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_main.c.d -g -O2 -Wall -W -Werror -c -o src/main.o src/main.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_args.c.d -g -O2 -Wall -W -Werror -c -o src/args.o src/args.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_ccache.c.d -g -O2 -Wall -W -Werror -c -o src/ccache.o src/ccache.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_cleanup.c.d -g -O2 -Wall -W -Werror -c -o src/cleanup.o src/cleanup.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_compopt.c.d -g -O2 -Wall -W -Werror -c -o src/compopt.o src/compopt.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_conf.c.d -g -O2 -Wall -W -Werror -c -o src/conf.o src/conf.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_counters.c.d -g -O2 -Wall -W -Werror -c -o src/counters.o src/counters.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_execute.c.d -g -O2 -Wall -W -Werror -c -o src/execute.o src/execute.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_exitfn.c.d -g -O2 -Wall -W -Werror -c -o src/exitfn.o src/exitfn.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_hash.c.d -g -O2 -Wall -W -Werror -c -o src/hash.o src/hash.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_hashutil.c.d -g -O2 -Wall -W -Werror -c -o src/hashutil.o src/hashutil.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_language.c.d -g -O2 -Wall -W -Werror -c -o src/language.o src/language.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_lockfile.c.d -g -O2 -Wall -W -Werror -c -o src/lockfile.o src/lockfile.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_manifest.c.d -g -O2 -Wall -W -Werror -c -o src/manifest.o src/manifest.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_mdfour.c.d -g -O2 -Wall -W -Werror -c -o src/mdfour.o src/mdfour.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_stats.c.d -g -O2 -Wall -W -Werror -c -o src/stats.o src/stats.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_unify.c.d -g -O2 -Wall -W -Werror -c -o src/unify.o src/unify.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_util.c.d -g -O2 -Wall -W -Werror -c -o src/util.o src/util.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_version.c.d -g -O2 -Wall -W -Werror -c -o src/version.o src/version.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_getopt_long.c.d -g -O2 -Wall -W -Werror -c -o src/getopt_long.o src/getopt_long.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_hashtable.c.d -g -O2 -Wall -W -Werror -c -o src/hashtable.o src/hashtable.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_hashtable_itr.c.d -g -O2 -Wall -W -Werror -c -o src/hashtable_itr.o src/hashtable_itr.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_murmurhashneutral2.c.d -g -O2 -Wall -W -Werror -c -o src/murmurhashneutral2.o src/murmurhashneutral2.c
gcc -DHAVE_CONFIG_H -DSYSCONFDIR=/usr/local/etc -I. -I./src -I./unittest  -MD -MP -MF .deps/src_snprintf.c.d -g -O2 -Wall -W -Werror -c -o src/snprintf.o src/snprintf.c
gcc -g -O2 -Wall -W -Werror -o ccache src/main.o src/args.o src/ccache.o src/cleanup.o src/compopt.o src/conf.o src/counters.o src/execute.o src/exitfn.o src/hash.o src/hashutil.o src/language.o src/lockfile.o src/manifest.o src/mdfour.o src/stats.o src/unify.o src/util.o src/version.o src/getopt_long.o src/hashtable.o src/hashtable_itr.o src/murmurhashneutral2.o src/snprintf.o   -lm  -lz
[91m+ make install
[0masciidoc -a revnumber=3.4.2 -d manpage -b docbook -o - doc/MANUAL.adoc | \
  perl -pe 's!<literal>(.*?)</literal>!<emphasis role="strong">\1</emphasis>!g' >doc/MANUAL.xml
a2x --doctype manpage --format manpage doc/MANUAL.xml
[91ma2x: ERROR: "xsltproc"  --stringparam callout.graphics 0 --stringparam navig.graphics 0 --stringparam admon.textlabel 1 --stringparam admon.graphics 0  "/etc/asciidoc/docbook-xsl/manpage.xsl" "/work/deps/ccache/doc/MANUAL.xml" returned non-zero exit status 5
[0mdev.mk:175: recipe for target 'doc/ccache.1' failed
[91mmake: *** [doc/ccache.1] Error 1
[0mThe command '/bin/sh -c /work/ubuntu_ccache.sh' returned a non-zero code: 2
Traceback (most recent call last):
  File "ci/build.py", line 347, in <module>
    sys.exit(main())
  File "ci/build.py", line 273, in main
    build_docker(platform, docker_binary, registry=args.docker_registry)
  File "ci/build.py", line 83, in build_docker
    check_call(cmd)
  File "/usr/lib/python3.5/subprocess.py", line 581, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['docker', 'build', '-f', 'docker/Dockerfile.build.jetson', '--build-arg', 'USER_ID=1001', '--cache-from', 'mxnetci/build.jetson', '-t', 'mxnetci/build.jetson', 'docker']' returned non-zero exit status 2
script returned exit code 1
```

--- a/ci/docker/install/ubuntu_ccache.sh
+++ b/ci/docker/install/ubuntu_ccache.sh
@@ -24,6 +24,12 @@ set -ex
 pushd .
 
 apt update
+apt install -y \
+    libxslt1-dev \
+    docbook-xsl \
+    xsltproc \
+    libxml2-utils
+
 apt install -y --no-install-recommends \
     autoconf \
     asciidoc \
@@ -45,7 +51,6 @@ cd ccache
 make -j$(nproc)
 make install
 
-cd /work/deps
 rm -rf /work/deps/ccache
 
 popd

